### PR TITLE
KAT disable

### PR DESCRIPTION
**Pull request by GermanHydrogen**
Added that all medic settings disable disable KAT functionallities beside hardcore.

If this is PR sufficient to achive that has to be tested with further players on our server.

### CODE REVIEW

File: addons/main/modKonfigurationsTemplates/medicSettings/kat-disable.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
+ * Should disable all KAT functionallity
+*/
+
+// KAT - ADV Medical: Airway
+force force kat_airway_enable = false;
+
+// KAT - ADV Medical: Breathing
+force kat_breathing_advPtxEnable = false;
```
Comment (TAINCER) [inline addons/main/modKonfigurationsTemplates/medicSettings/kat-disable.h:9]: Intentionally not forcing for the mission?

**Review by TAINCER — CHANGES_REQUESTED** [review]

File: addons/main/modKonfigurationsTemplates/medicSettings/kat-disable.h (hunk 1 above)
Comment (GermanHydrogen) [reply addons/main/modKonfigurationsTemplates/medicSettings/kat-disable.h]: Should be fixed now

**Comment by TAINCER** [comment]
@GermanHydrogen Please merge [74d4cf1](https://github.com/Rosenrudel/RR_Commons_Resources/pull/19/commits/74d4cf1c852f89d080def19ea4a484ce6c8e95f8) with [d1fb261](https://github.com/Rosenrudel/RR_Commons_Resources/pull/19/commits/d1fb261a2f398b04359e8c31efddecb0299199e5)

Also maybe consider adding the the comments in [244b044](https://github.com/Rosenrudel/RR_Commons_Resources/pull/19/commits/244b0448f91273a190a2aa6d539b6fdd10919325) to [d1fb261](https://github.com/Rosenrudel/RR_Commons_Resources/pull/19/commits/d1fb261a2f398b04359e8c31efddecb0299199e5) aswell.

**Review by TAINCER — APPROVED** [review]
LGTM

**Pull request merged**